share/spack/gitlab/cloud_e4s_pipelines/stacks/e4s/spack.yaml

```diff
@@ -88,10 +88,10 @@ spack:
     - raja
     - rempi
     - scr
-    - slate ^openblas@0.3.5 threads=openmp ^cuda@10.2.89
+    - slate ^openblas@0.3.6 threads=openmp ^cuda@10.2.89
     - slepc
     - stc
-    - strumpack ~slate
+    - strumpack ~slate ^openblas@0.3.6 threads=openmp
```

Comment:
@eugeneswalker I had to add this today after rebasing on spack `develop`.  Can you point me at your E4S source of truth repo again?  I just want to check we're not drifting too far off from what you have.

Reply:
This is as close as there is to a public, source-of-truth repo for the develop, non-tagged, E4S:
https://github.com/spack/spack-configs/tree/master/UOREGON/E4S-Develop

I made the exact same explicit constraint earlier today as well.

Reply:
> This is as close as there is to a public, source-of-truth repo for the develop, non-tagged, E4S:
> https://github.com/spack/spack-configs/tree/master/UOREGON/E4S-Develop

Cool, thank you 👍 

> I made the exact same explicit constraint earlier today as well.

Good to know, thanks.